internals/write_creator.py

```diff
@@ -0,0 +1,15 @@
+from framework.basehandlers import FlaskHandler
+from internals.models import Feature
+
+
+class UpdateCreatorHandler(FlaskHandler):
+  def get_template_data():
+    """Writes string creator field from created_by user field."""
+    q = Feature.query()
+    features = q.fetch()
+    for feature in features:
```

Comment:
This would be more robust if could make incremental progress when run repeatedly.   I don't think that we have enough features in datastore to make this timeout, but just in case, let's code it to handle that.

I think writing it this way would allow us to run repeatedly with the code quickly skipping over Features that have already been updated:
```
for feature in features:
  if feature.created_by and not feature.creator:
    feature.creator = feature.created_by.email()
    feature.put()
```
Note that I am calling feature.created_by.email() because str(feature.created_by) ends up calling feature.created_by.nickname() which leaves off "@gmail.com".